app/helpers/days_helper.rb

```diff
@@ -19,4 +19,13 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def total_cohort_students
+    @total ||= cohort.students.length
+  end
+
+  def completed_students(activity)
+    completed = User.count_by_sql ["SELECT count(*) FROM users JOIN activity_submissions on activity_submissions.user_id = users.id WHERE cohort_id = ? AND activity_submissions.activity_id = ?", cohort.id, activity.id]
```

Comment:
This is a nicely precise query. I don't think queries should be in a helper class, though. Can you please move this to the model?